inspirehep/modules/orcid/utils.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from .schema import orcid_overdo
+
+
+def convert_to_orcid(record):
+    orcid_json = orcid_overdo.do(record)
+    try:
+        if record.get('collection') is 'published':
+            date = record.get('imprints').get('date')
+            date = date.split('-')
+            final_date = ['', '', '']
+            if date[0]:
+                final_date[0] = date[0]
+            if date[1]:
+                final_date[1] = date[1]
+            if date[2]:
+                final_date[2] = date[2]
+            publication_date = {'year': final_date[0],
+                                'month': final_date[1],
+                                'day': final_date[2]
+                                }
+            orcid_json['publication-date'] = publication_date
```

Comment:
I think it possible to move that into `dojson` as well. I have to check how but there are some tricks available.